### PR TITLE
Twitter Data Source: switch to use rtweet package

### PR DESCRIPTION
**Pull request by hidekoji**
### Description

switch to user rtweet package

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -749,11 +749,11 @@ queryODBC <- function(dsn,username, password, additionalParams, numOfRows = 0, q
...
 #' @export
 getTwitter <- function(n=200, lang=NULL,  lastNDays=30, searchString, tokenFileId=NULL, withSentiment = FALSE, ...){
-  if(!requireNamespace("twitteR")){stop("package twitteR must be installed.")}
+  if(!requireNamespace("rtweet")){stop("package rtweet must be installed.")}
   loadNamespace("lubridate")
-
   twitter_token = getTwitterToken(tokenFileId)
-  twitteR::use_oauth_token(twitter_token)
+  twitter_token = rtweet:::check_token(twitter_token);
+  browser()
```
Comment (hideaki) [inline R/system.R]: browser() should be removed.

File: R/system.R [hunk 2]
```diff
@@ -768,19 +768,33 @@ getTwitter <- function(n=200, lang=NULL,  lastNDays=30, searchString, tokenFileI
...
-  if(length(tweetList)>0){
-    ret <- twitteR::twListToDF(tweetList)
-    if(withSentiment){
-      # calculate sentiment
-      ret %>% dplyr::mutate(sentiment = get_sentiment(text))
-    } else {
-      ret
-    }
+  tweetList <- rtweet::search_tweets(q = searchString, token = twitter_token, n = n, lang = lang, verbose = TRUE, since = since, unitl = until, locale = locale, geocode = geocode, type = resultType,  retryonratelimit=TRUE)
+  #tweetList <- rtweet::search_tweets(q = searchString,n = n,type = resultType, retryonratelimit=TRUE)
```
Comment (hideaki) [inline R/system.R]: I guess this line can be removed.

File: R/system.R [hunk 3]
```diff
@@ -768,19 +768,33 @@ getTwitter <- function(n=200, lang=NULL,  lastNDays=30, searchString, tokenFileI
...
+    # calculate sentiment
+    tweetList %>% dplyr::mutate(sentiment = get_sentiment(text))
   } else {
-    stop('No Tweets found.')
+    tweetList
+  }
+
+}
+
+twitterListToDF <- function(twList) {
```
Comment (hideaki) [inline R/system.R]: Looks this function is not used?

Comment (hidekoji) [reply R/system.R]: removed it.

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R]: removed it.

File: R/system.R [hunk 4]
```diff
@@ -768,21 +767,15 @@ getTwitter <- function(n=200, lang=NULL,  lastNDays=30, searchString, tokenFileI
...
-  if(length(tweetList)>0){
-    ret <- twitteR::twListToDF(tweetList)
-    if(withSentiment){
-      # calculate sentiment
-      ret %>% dplyr::mutate(sentiment = get_sentiment(text))
-    } else {
-      ret
-    }
+  tweetList <- rtweet::search_tweets(q = searchString, token = twitter_token, n = n, lang = lang, verbose = TRUE, since = since, unitl = until, locale = locale, geocode = geocode, type = resultType,  retryonratelimit=TRUE)
+  if(withSentiment){
```
Comment (hideaki) [inline R/system.R]: Looks the handling of "no tweets found" case is removed, but is it ok? if so why?

**Pull request merged**